VagrantProvision.sh

```diff
@@ -204,7 +216,7 @@ if [ ! -f bin/osmosis ]; then
     ln -s $HOME/bin/osmosis_src/bin/osmosis $HOME/bin/osmosis
 fi
 
-if ! ogrinfo --formats | grep --quiet FileGDB; then
+if ! hash ogrinfo >/dev/null 2>&1 || ogrinfo --formats | grep --quiet FileGDB; then
```

Comment:
This is a quick check to see if `ogrinfo` is installed before trying to execute it.  `hash` returns 0 (success) if the executable is found in the path.